pom.xml

```diff
@@ -217,7 +217,7 @@
         <version.org.wildfly.arquillian>2.1.0.Alpha1</version.org.wildfly.arquillian>
         <version.org.wildfly.build-tools>1.2.2.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>3.0.0.Beta24</version.org.wildfly.core>
+        <version.org.wildfly.core>3.0.0.Beta25</version.org.wildfly.core>
```

Comment:
No. If you rebase your PR you will see that core is already on Beta26. Your core PR will be part of a later core release